crates/astria-composer/src/collectors/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod geth;
```

Comment:
Rexport `geth::Geth` here, so that we can refer to it as `collectors::Geth` elsewhere